srcpkgs/linux-steam-integration/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'linux-steam-integration'
+pkgname=linux-steam-integration
+version=0.7.3
+revision=1
+archs="x86_64 i686"
+build_style=meson
+configure_args="-Dwith-shim=co-exist \
+    -Dwith-steam-binary=/usr/bin/steam \
+    -Dwith-new-libcxx-abi=true \
+    -Dwith-libressl-mode=native \
+    -Dwith-frontend=true"
+hostmakedepends="pkg-config"
+makedepends="gtk+3-devel"
+depends="gtk+3 glib glibc steam"
```

Comment:
The first three should be detected automatically once the package is built. I'm not sure this should depend on steam either... Couldn't it still be useful if you use flatpak or install it manually some other way?

Reply:
It is built to assume that normal steam is at `/usr/bin/steam`, so if flatpak steam follows that, yeah, I guess it'd work